harvester/harvester
# bump go mod to fix some dep issues

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
 go mod contains some dependency issues.

**Solution:**
bump go mod to fix those dependency issues.

**Related Issue:**
https://github.com/harvester/security/issues/23

**Test plan:**
- build the latest Harvester API server.
- replace with the custom image in any v1.1.2.x cluster.
- make sure it runs correctly and contains no extra error logs.
